### PR TITLE
Support Kinesis stream events 

### PR DESCRIPTION
**Pull request by ngonzalezromero**
Support for Kinesis stream event types using the @Kinesis decorator.

### CODE REVIEW

**Review by michaelCastillo — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "lambaa",
-    "version": "3.2.1",
```
Comment (jackholmes1010) [inline package-lock.json]: Do you need to update the package version in `package.json`?

**Review by jackholmes1010 — COMMENTED** [review]
looks good. Could you also run the formatter, `npm run prettier` please.

**Review by jackholmes1010 — COMMENTED** [review]
would you also mind adding a little example here: https://github.com/vacasaoss/lambaa/blob/master/README.md#controllers

**Review by jackholmes1010 — APPROVED** [review]

File: tests/testUtil.ts [hunk 2]
```diff
@@ -134,6 +135,29 @@ export const createDynamoDbStreamEvent = (
...
+        eventID:
+            "shardId-000000000000:49545115243490985018280067714973144582180062593244200961",
+        invokeIdentityArn: "arn:aws:iam::EXAMPLE",
+        eventVersion: "1.0",
+        eventName: "aws:kinesis:record",
+        eventSourceARN: `${tableArn}`,
+        awsRegion: "us-east-1",
+    })),
+})
+
```
Comment (johnmikk) [inline tests/testUtil.ts:160]: Is the data in this test event sensitive in any way?  It looks like it might be copied from a real event.

Comment (ngonzalezromero) [reply tests/testUtil.ts:160]: Hi @johnmikk . the data was copied from test data generated in AWS console. Is not a real data

**Review by johnmikk — APPROVED** [review]
Looks good 👌

**Pull request merged**